tests/stress_tests/common/utils.cpp

```diff
@@ -25,11 +25,12 @@ std::string fileNameNoExt(const std::string &filepath) {
 
 
 static size_t parseLine(char* line) {
-    // This assumes that a digit will be found and the line ends in " Kb".
     size_t i = strlen(line);
     const char* p = line;
     while (*p <'0' || *p > '9') p++;
-    line[i-3] = '\0';
+    if (line[i-1] == 'b')
```

Comment:
it should be checking for both K and b and that the length > 2

Reply:
Done